### PR TITLE
Delay the Get-AzAccessToken Breaking Change

### PR DESCRIPTION
**Pull request by msJinLei**
## Description
image

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Accounts/Accounts/help/Get-AzAccessToken.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ schema: 2.0.0
 # Get-AzAccessToken
 
 ## SYNOPSIS
-Get raw access token. When using -ResourceUrl, please make sure the value does match current Azure environment. You may refer to the value of `(Get-AzContext).Environment`.
+Get secure raw access token. When using -ResourceUrl, please make sure the value does match current Azure environment. You may refer to the value of `(Get-AzContext).Environment`.
+> **_NOTE:_**  The current default output type `PSAccessToken` is going to be deprecate. Please always use `-AsSecureString` to avoid the upcoming breaking change. After that the output turns to be `PSSecureAccessToken`, which contains `token` as a `SecureString`. Please refer to [Unprotect-SecureString](https://github.com/Azure/azure-powershell/blob/main/src/Aks/Aks.Autorest/utils/Unprotect-SecureString.ps1) for how to convert from `SecureString` to plain text.
```
Comment (isra-fel) [inline src/Accounts/Accounts/help/Get-AzAccessToken.md:12]: It's unlikely that we will maintain this link in the future. My suggestion is contribute to our official FAQ https://learn.microsoft.com/en-us/powershell/azure/faq?view=azps-12.4.0

Comment (isra-fel) [reply src/Accounts/Accounts/help/Get-AzAccessToken.md:12]: Probably needs some rewriting. The change of the Token type is more important than the output type. I suggest not mentioning the output type.

**Review by isra-fel — CHANGES_REQUESTED** [review]
Added inline comments. Please also consider modifying the description of `-AsSecureString` param

File: src/Accounts/Accounts/help/Get-AzAccessToken.md (hunk 1 above)
Comment (msJinLei) [reply src/Accounts/Accounts/help/Get-AzAccessToken.md]: Done

**Review by isra-fel — DISMISSED** [review]

**Review by vidai-msft — APPROVED** [review]

**Pull request merged**